### PR TITLE
feat: Aurora MySQL TLS connections always enabled

### PR DESCRIPTION
**Pull request by zucchinidev**
Using the new TF provider, by default, all users will connect using TLS due to the configuration per-account level.

[#183650281](https://www.pivotaltracker.com/story/show/183650281)

### Checklist:

* [x] Have you added Release Notes in the docs repositories?
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

File: terraform/aurora-mysql/bind/versions.tf [hunk 1]
```diff
@@ -4,6 +4,10 @@ terraform {
       source  = "hashicorp/mysql"
```
Comment (FelisiaM) [inline terraform/aurora-mysql/bind/versions.tf:4]: Is this still required?

Comment (zucchinidev) [reply terraform/aurora-mysql/bind/versions.tf:4]: Good question @FelisiaM, I initially thought it wasn’t necessary, but then I changed opinion understanding that they can have applications with the service in beta. Do you think we should eliminate and simplify it? After all, it’s a service that’s supposed to be in beta. I’m waiting for your answer to eliminate it.

Comment (FelisiaM) [reply terraform/aurora-mysql/bind/versions.tf:4]: I worry about what the plan will be in the future to remove it and will we get stuck with it for a long time.
Our options are
1. Remove it now and see if anyone actually was using it. If they complain we put it back as a result.
2. Leave it in indefinitely (ideally until we know everyone has upgraded)

Given our current knowledge about the usage of this service, I would go with the riskier option 1.

**Review by FelisiaM — APPROVED** [review]
👍

**Pull request merged**